src/desktop/apps/article/routes.ts

```diff
@@ -144,7 +144,8 @@ export async function index(req, res, next) {
     res.locals.sd.RESPONSIVE_CSS = createMediaStyle()
 
     // CollectionsRail a/b test
-    const hasCollectionsRail = EDITORIAL_COLLECTIONS_RAIL.toString() === "1"
+    const hasCollectionsRail =
+      EDITORIAL_COLLECTIONS_RAIL === "1" || EDITORIAL_COLLECTIONS_RAIL === 1
```

Comment:
`toString()` was causing some issues so I went ahead and did an explicit check for the integer 1.